pkg/apis/3scale/v1alpha1/apimanager_types.go

```diff
@@ -2,17 +2,35 @@ package v1alpha1
 
 import (
 	"fmt"
-
 	"github.com/RHsyseng/operator-utils/pkg/olm"
 
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
this should stay named `corev1` with our new import naming convention

Reply:
@davidffrench is this convention defined anywhere? Want to make sure I'm also following it correctly

Reply:
where is that naming convention defined?

This is a copy/paste from the 3scale operator. I tried to add 3scale-operator as a dependency so that we don't need to duplicate the api definitions, but unfortunately it doesn't work. The version of the operator-sdk it's using is just too old. Hopefully with Sergio's work, we will be able to get rid of this from the integreatly operator completely.

Given that this is a copy/paste code and it is something we want to get rid of, I don't think it worth the effort to rename it.

Reply:
Good questions. This was recently done by @grdryn in a couple of PRs. @grdryn Is the naming convention documented anywhere?

Reply:
@davidffrench this file is removed now.